src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -376,11 +376,18 @@ struct SummaryConfigContext {
         // keyword that does not specify any nodes (e.g., "GPR /"), and to
         // check for missing nodes if a keyword is erroneously specified.
 
-        return std::any_of(sect.begin(), sect.end(),
-            [](const DeckKeyword& keyword)
-        {
-            return is_node_keyword(keyword.name());
-        });
```

Comment:
Does this mean we can't use standard algorithms with the new iterator?

Reply:
No - it means we *could not* at a stage in the development - I'll revert that change